modules/ROOT/pages/osv2-usage.adoc

```diff
@@ -8,17 +8,40 @@ endif::[]
 
 You can view a chart indicating how many API requests have been consumed.
 
-* Every API call (via the connector or API) to Object Store v2 counts as one API request.
+* Only events count towards your monthly quota.
 +
-All requests to the broker API, such as creating and deleting objects, are counted against the quota.
-* Up to 100 KB of object-payload data can be transferred in a single API call,
-which bills as one request.
-* Each additional 100 KB of payload data is charged as an additional API request. 
-* Monthly billing usage information is delayed by 24 hours.
+Each event (such as create, modify, or delete) on a store or an item counts as one billable unit against the quota.
++
+See <<billevents>>.
+* For each billable unit, up to 100 KB of object-payload data counts as one request against the quota.
```

Comment:
For some events payload might not be applicable. May be, change it to

For each billable unit, up to 100 KB of object-payload data (where applicable) counts as one request against the quota.